content/fundamental-redirects.js

```diff
@@ -0,0 +1,1145 @@
+const startRe = /^\^\/?/;
+const startTemplate = /^\//;
+
+function redirect(pattern, template, options = {}) {
```

Comment:
The `options` are never used, so we're never passing along, in the result, the `permanent` option. Currently, the user of `resolveFundamental()` has no idea whether the redirect, if there is one, is permanent or not (since some of these fundamental redirects are not permanent). So should we consider returning an object here (with a `permanent` property) rather than a string, or perhaps a tuple of `[redirect, permanent]`?

Reply:
@peterbe and I had a conversation about this on Slack. We may be able to make the case that all of the redirects that are currently 302's could be converted to 301's, but I still think we should support both cases for the future. For example, there are times when you would like to start with 302's but then convert to 301's later.

Reply:
I would recommend that this function return an object, with two properties: `url` (or `location`), and `permanent` (or maybe the actual status code, `301` or `302`, instead?).